File: app/src/main/java/bobrchess/of/by/belaruschess/presenter/callback/CallBackAddTournament.java
package bobrchess.of.by.belaruschess.presenter.callback;

import java.util.List;

import bobrchess.of.by.belaruschess.dto.CountryDTO;
import bobrchess.of.by.belaruschess.dto.PlaceDTO;
import bobrchess.of.by.belaruschess.dto.RankDTO;
import bobrchess.of.by.belaruschess.dto.TournamentDTO;
import bobrchess.of.by.belaruschess.dto.UserDTO;

/**
 * Created by Igor on 11.04.2018.
 */

public interface CallBackAddTournament extends CallBack {
    void onResponse(TournamentDTO tournamentDTO);
    void onRefereeResponse(List<UserDTO> referees);
    void onPlaceResponse(List<PlaceDTO> places);
}

File: app/src/main/java/bobrchess/of/by/belaruschess/presenter/callback/CallBackAddTournament.kt
package bobrchess.of.by.belaruschess.presenter.callback

import bobrchess.of.by.belaruschess.dto.PlaceDTO
import bobrchess.of.by.belaruschess.dto.TournamentDTO
import bobrchess.of.by.belaruschess.dto.UserDTO

/**
 * Created by Igor on 11.04.2018.
 */

interface CallBackAddTournament : CallBack {
    fun onResponse(tournamentDTO: TournamentDTO)
    fun onRefereeResponse(referees: List<UserDTO>)
    fun onPlaceResponse(places: List<PlaceDTO>)
}
